lib/live_debugger/live_views/sidebar_live.ex

```diff
@@ -104,12 +112,49 @@ defmodule LiveDebugger.LiveViews.SidebarLive do
           node_id={@node_id}
           highlight?={@highlight?}
           parent_lv_process={@parent_lv_process}
+          memory_raw_kb={@memory_raw_kb}
+          memory_gc_kb={@memory_gc_kb}
+          assigns_memory_kb={@assigns_memory_kb}
         />
       </.sidebar_slide_over>
     </div>
     """
   end
 
+  @impl true
+  def handle_info(:assign_memory_usage, socket) do
+    Process.send_after(self(), :assign_memory_usage, 2000)
+
+    raw =
+      case :erlang.process_info(self(), :memory) do
```

Comment:
`self()` in this context returns a PID of `SidebarLive` LiveView - not debugged LiveView process

Reply:
My bad @kraleppa wasn't thinking - targeting the `lv_process.root_pid` which i believe will be the correct one now.